src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -209,6 +209,12 @@
   (not (empty? (jdbc/query ["SELECT 1 FROM certnames WHERE certname=? LIMIT 1"
                             certname]))))
 
+(defn certname-metadata
+  [certname]
+  {:pre [certname]}
+  (jdbc/query ["select certname, expired, deactivated from certnames where certname=?"
+               certname]))
```

Comment:
should we just call `first` here since this will never return more than one row?